public/data/hailer24.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Anmol Bhutani",
+  "type": "personal",
+  "bio": "Just trying to git gud",
+  "avatar": "https://github.com/hailer24.png",
+  "links": [
+    {
+      "name": "Connect with me on GitHub",
+      "url": "https://github.com/hailer24",
+      "icon": "github"
+    },
+    {
+      "name": "Conenct with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/anmol-bhutani-b132931a5/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Take a look at my portfolio",
+      "url": "https://mole24.netlify.app/",
+      "icon": "link"
+    },
+    {
+      "name": "Check my Codeforces account",
+      "url": "https://codeforces.com/profile/mole24",
+      "icon": "codeforces"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Web Development",
+      "date": "June 2021",
+      "icon": "Globe",
```

Comment:
Shouldn't we use `globe` here without capitalizing it?

Reply:
changed to lowercase